src/pages/home/report/ReportActionsView.js

```diff
@@ -1,10 +1,13 @@
 import React from 'react';
-import {View, Keyboard, AppState} from 'react-native';
+import {
+    Animated, View, Keyboard, AppState, Easing,
+} from 'react-native';
```

Comment:
```suggestion
import {
    Animated, 
    View, 
    Keyboard, 
    AppState, 
    Easing,
} from 'react-native';
```

Based on an internal discussion it looks like we'd prefer imports of about 5+ to be split into their own lines.

Reply:
Got it!